### ISSUE TITLE
Node Suite not importing after update

### ISSUE THREAD

**Issue by GoldmanOz** [WASasquatch/was-node-suite-comfyui#168]
I'm running an M1 MacBook Pro and have had no issue with Was Node Suite importing until the last update I did, all my installs have been done through the Manager and until the last I have had no issues...Everything in manager is up to date and telling me all is installed but I'm having problems with this Suit and the Impact Pack now, I know this is not the place to ask about the Impact Pack but thought it might help knowing both will not import now and both are showing as properly installed.. ???

Any ideas would be greatly appreciated...I have done nothing to the original installs other than the Manager updates..

**Comment by WASasquatch** [comment]
You should be receiving an error in your console if WAS Node Suite wasn't properly loaded. That would be the key to figuring what's wrong.

**Comment by GoldmanOz** [comment]
This is the only error I can seem to find....???
<img width="756" alt="Was2" src="https://github.com/WASasquatch/was-node-suite-comfyui/assets/139660390/2eac9036-4c5d-42db-b5ba-d328d8568d57">

**Comment by WASasquatch** [comment]
Looks like you need to run requirements again

**Comment by GoldmanOz** [comment]
I navigated to the W-N-S in my terminal and ran 'pip install -r requirements.txt' there...no change when loading Comfy after that..any other possible ideas ???  
As I said previous it has been working prior to the last update I did via Manager..

**Comment by WASasquatch** [comment]
Yeah, Data It has been making changes to ComfyUI Manager, which is not my project.

**Comment by GoldmanOz** [comment]
No change unfortunately...installed opencv-python, already satisfied...ComfyUI depends so much on your Suit and the Impact Pack  it's now lost most of its functionality for me...Not your fault I know but I don't know what to do from here..

**Comment by WASasquatch** [comment]
You're on linux?

**Comment by kronkinatorix** [comment]
I'm having the same issue on linux mint 21 victoria. 

trying to pip install opencv-python-headless[ffmpeg]==4.8.0.76"  displays:

Requirement already satisfied: opencv-python-headless[ffmpeg]==4.8.0.76 in .........ComfyUI/venv/lib/python3.10/site-packages (4.8.0.76)
WARNING: opencv-python-headless 4.8.0.76 does not provide the extra 'ffmpeg'
Requirement already satisfied: numpy>=1.17.3 in /............omfyUI/venv/lib/python3.10/site-packages (from opencv-python-headless[ffmpeg]==4.8.0.76) (1.23.5)

uninstalling all install opencv and reinstalling does not fix the issue.

**Comment by kronkinatorix** [comment]
for anyone else that has the same issue: installing 

pip install opencv-python-headless[ffmpeg]==4.8.0.74 appears to have fixed the issue.

**Comment by WASasquatch** [comment]
So appears to be a change in recent builds of opencv, at least in Linux.
